src/main/java/com/github/mc1arke/sonarqube/plugin/CommunityBranchPlugin.java

```diff
@@ -67,8 +71,80 @@ public void load(CoreExtension.Context context) {
                 //the name and description shown on the UI are automatically loaded from core.properties so don't need to be specified here
                 PropertyDefinition.builder(CoreProperties.LONG_LIVED_BRANCHES_REGEX).onQualifiers(Qualifiers.PROJECT)
                         .category(CoreProperties.CATEGORY_GENERAL).subCategory(CoreProperties.SUBCATEGORY_BRANCHES)
-                        .defaultValue(CommunityBranchConfigurationLoader.DEFAULT_BRANCH_REGEX).build());
+                        .defaultValue(CommunityBranchConfigurationLoader.DEFAULT_BRANCH_REGEX).build(),
+
+                PropertyDefinition.builder("sonar.pullrequest.provider").category(PULL_REQUEST_CATEGORY_LABEL)
+                        .subCategory(GENERAL)
+                        .onlyOnQualifiers(Qualifiers.PROJECT).name("Provider").type(PropertyType.SINGLE_SELECT_LIST)
+                        .options("GitlabServer").build(),
+
+                PropertyDefinition.builder("sonar.pullrequest.base.image.url")
+                        .category(PULL_REQUEST_CATEGORY_LABEL)
+                        .onlyOnQualifiers(Qualifiers.PROJECT)
+                        .name("Base image URL")
+                        .defaultValue("https://raw.githubusercontent.com/mc1arke/sonarqube-community-branch-plugin/pr-decoration/src/main/resources/pr-decoration-images")
+                        .description("Default: https://raw.githubusercontent.com/mc1arke/sonarqube-community-branch-plugin/master/src/main/resources/pr-decoration-images")
+                        .type(PropertyType.STRING)
+                        .build(),
```

Comment:
See https://github.com/mc1arke/sonarqube-community-branch-plugin/pull/31#discussion_r362045510 - you shouldn't need to define this as a property, unless there was something you were specifically trying to do?

Reply:
removed.